server/__init__.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+from flask import Flask, request, send_file, session, jsonify
+from server.settings import *
+
+
+def create_app(test_config=None):
+    app = Flask(__name__)
+
+    validate_server_settings()
+
+    if test_config is not None:
+        app.config.from_mapping(test_config)
+    
+    from .home import views as home_views
+    app.register_blueprint(home_views.bp)
```

Comment:
`home/views.py` serves various entry points to React. It renders an HTML template along with the javascript bundle from babel/webpack.